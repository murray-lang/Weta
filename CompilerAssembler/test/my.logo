bytearray [txbuff 5]
aset :txbuff 0 0
aset :txbuff 1 1
aset :txbuff 2 2
aset :txbuff 3 3
aset :txbuff 4 4

send 42
send 0 "txbuff 5
