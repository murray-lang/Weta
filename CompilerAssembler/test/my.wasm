	.global _start

	
	.data
txbuff:
	.rept 5
		.byte		0
	.endr
	
	.text
_start:
	uint8		0
	int16		0
	global		txbuff
	aubset
	uint8		1
	int16		1
	global		txbuff
	aubset
	uint8		2
	int16		2
	global		txbuff
	aubset
	uint8		3
	int16		3
	global		txbuff
	aubset
	uint8		4
	int16		4
	global		txbuff
	aubset
	int16		42
	stx
	uint8		5
	global		txbuff
	uint8		0
	txn
	uint8	1
	pop
	exit
	
	.end
