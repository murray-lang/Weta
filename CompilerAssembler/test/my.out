{"address":0,"codes":[1,99,59,2,0,42,149,2,0,142,149,2,0,54,150,2,0,154,150,2,0,90,151,2,0,190,151,2,0,180,152,2,1,24,152,154,153,154,153,96]}